src/routes/[type]/[id]/+page.ts

```diff
@@ -44,6 +44,6 @@ export const load = (async ({ fetch, params }) => {
 		mediaType: type,
 		mediaID: id,
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
-		db: data as any
+		riven: data as any
```

Comment:
Not related to the pr, but just a general nitpick. Why not use types here? At least for the fields we use.

Reply:
because typescipt allows us to declare any, why bother? Ill fix that real quick.